controller/replicator.go

```diff
@@ -505,3 +506,27 @@ func (r *replicator) GetCloneStatus(address string) (string, error) {
 
 	return status, nil
 }
+
+func (r *replicator) GetPreloadStatus(address string) error {
+	backend, ok := r.backends[address]
+	if !ok {
+		return fmt.Errorf("Cannot find backend %v", address)
+	}
+	for true {
+		status, err := backend.backend.GetPreloadStatus()
```

Comment:
can we avoid using `backend` here?

Reply:
We can't, this is the way, we can get the PreloadStatus.